hepcrawl/spiders/elsevier.py

```diff
@@ -0,0 +1,235 @@
+import glob
+import os
+import shutil
+import tempfile
+import xml.etree.ElementTree as et
+import zipfile
+from io import BytesIO
+
+import boto3
+import scrapy
+from scrapy import Request, Spider
+
+from . import StatefulSpider
+from ..parsers import ElsevierParser
+from ..utils import ParsedItem, strict_kwargs
+
+
+class ElsevierSpider(StatefulSpider):
```

Comment:
does it need to be a `StatefulSpider`? if you're checking manually which zip files we've already received, I'm not sure there's much point to it.